.eslintrc

```diff
@@ -1,7 +1,10 @@
 {
   "parser": "babel-eslint",
-  "extends": "airbnb",
-  "plugins": ["flowtype", "flowtype-errors", "prettier"],
+  "extends": [
+    "airbnb",
+    "prettier"
```

Comment:
Several things here; this is a nice change, but we should use the following config too to ensure smooth compatibility with flow: `"prettier/flowtype"`

Please do not remove the prettier plugin (see below)